c31-shadowlyn/react-nutshell-shadowlyn
# Put data in our database that we can use to move forward.

# Description

Here is the draft of our database.

Applies to issue # Database Creation

## Type of change

Please delete options that are not relevant.




# Testing Instructions

1. git fetch --all
2. git checkout cm-database
3. code .
4. check database.json.example
5. copy over content into database.json


# Checklist:

- [ x] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ x] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
